packages/theming/theme-tokens/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "@uifabricshared/build-native/tsconfig.json",
   "compilerOptions": {
     "outDir": "lib",
+    "resolveJsonModule": true,
```

Comment:
@rurikoaraki Talking to @afoxman about this, and we need this flag to import json properly